examples/scaling-criteo/03-Training-with-FastAI.ipynb

```diff
@@ -113,7 +113,7 @@
     "BASE_DIR = os.environ.get(\"BASE_DIR\", \"/raid/data/criteo\")\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    input_path = os.environ.get("INPUT_DATA_DIR", os.path.join(BASE_DIR, "test_dask/output"))
</pre><p>uses <code>os.path.join</code> and assumes <code>/</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>